openfisca_web_api/app.py

```diff
@@ -73,7 +73,16 @@ def get_root():
 
     @app.route('/parameters')
     def get_parameters():
-        return jsonify(data['parameters_overview'])
+        parameters = {
+            parameter['id']: {
+                'description': parameter['description'],
+                'href': '{}parameter/{}'.format(request.host_url, parameter['id'])
```

Comment:
We want to use `name` here, to get a link towards `http://localhost/parameter/taxes/income_tax_rate` instead of `http://localhost/parameter/taxes.income_tax_rate`